src/spacy_affixes/utils.py

```diff
@@ -87,7 +87,7 @@ def load_lexicon(lang="es", version="4.1"):
 def download_affixes(lang="es", version="4.1"):
     sys.stdout.write(f"Downloading affixes {lang}-{version}...\n")
     url = (f"https://raw.githubusercontent.com/TALP-UPC/FreeLing/"
-           f"{version}/data/{lang}/afixos.dat")
+           f"v{version}/data/{lang}/afixos.dat")
```

Comment:
How is it that it's been working before? Was that a Freeling or GitHub change?

Reply:
both work IIRC

Reply:
They do not.
- https://raw.githubusercontent.com/TALP-UPC/FreeLing/4.0/data/es/afixos.dat works
- https://raw.githubusercontent.com/TALP-UPC/FreeLing/v4.0/data/es/afixos.dat does not

Better to leave that `v` out for now. It works for 4.1 because is both a branch `v4.1` and a tagged commit `4.1`.